server/api/users.js

```diff
@@ -131,14 +131,22 @@ router.post("users.promote", auth(), async (ctx) => {
 router.post("users.demote", auth(), async (ctx) => {
   const userId = ctx.body.id;
   const teamId = ctx.state.user.teamId;
+  let { to } = ctx.body;
+
   const actor = ctx.state.user;
   ctx.assertPresent(userId, "id is required");
 
+  const possibleTos = ["member", "viewer"];
+  if (possibleTos.findIndex((possibleTo) => possibleTo === to) === -1) {
+    to = "member";
+  }
```

Comment:
Yes! this is good for now.

Reply:
@tommoor The fallback is `viewer` here. Previously `users.demote` would demote to `member`, Do you suggest changing the ternary to make `member` a fallback? 

We have a test failing because of this, I am confused between changing the testcase or the code 😅

Reply:
I'm not against making the default "member" if that helps reduce test churn